### DISCUSSION TITLE
How to add Custom modal instead of default asset manager in grapejs

### DISCUSSION THREAD

**Discussion by arjavsethi** [GrapesJS/grapesjs#4583] [Q&A] [answered]
I am integrating grapesjs builder with my nextjs website , i already have asset manager built previously. What I want is to pop up my modal instead of default modal of grapejs.
for code reference
[](https://stackoverflow.com/questions/73708657/how-to-add-custom-modal-instead-of-default-asset-manager-in-grapejs)

**Comment by artf** [comment, marked as answer]
You can see the example of custom modal here https://grapesjs.com/docs/modules/Modal.html#custom-modal
